local/new.go

```diff
@@ -45,6 +46,7 @@ func NewImage(repoName string, dockerClient DockerClient, ops ...ImageOption) (*
 		docker:           dockerClient,
 		repoName:         repoName,
 		inspect:          inspect,
+		history:          make([]v1.History, len(inspect.RootFS.Layers)),
```

Comment:
TODO: we should grab the actual history and only zero it out if the length of the history doesn't match the length of the layers